src/store.js

```diff
@@ -61,7 +67,7 @@ function localStore(config, emitter, store) {
     return keys.map((id, i) => {
       if (storeSize + i + 1 > config.storeOptions.recordLimit) {
         emitter.emit('cacheFull', { reason: 'Too many records', current: config.storeOptions.recordLimit, limit: config.storeOptions.recordLimit });
-        return;
+        return false;
```

Comment:
Like we discussed, this function will return a mix of `Set` and `boolean` which is not was its intended for.
Replace 
`return store.set(recordKey(id), value);` 
with 
```
store.set(recordKey(id), value);
return true;
```